lib/internal/modules/esm/hooks.js

```diff
@@ -537,15 +555,18 @@ class HooksProxy {
     }
   }
 
-  async makeAsyncRequest(method, ...args) {
+  async makeAsyncRequest(method, args, transferList) {
```

Comment:
Uncertain if `(method, args, transferList)` or `(method, transferList, ...args)` is better? Or if there is any performance difference between the two?

Reply:
Well, they're quite different in their effect: …args means anything after the previous is ported wholesale (and it doesn't care).

I think here, we would probably want `transferList, ...args`